permissions/plugin-validating-email-parameter.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "validating-email-parameter"
+github: "jenkinsci/validating-email-parameter-plugin"
+paths:
+- "org/jenkins-ci/plugins/validating-email-parameter"
```

Comment:
This should be your groupId + artifactId in path form. Your groupId is `io.jenkins.plugins`, so you need "io/jenkins/plugins/validating-email-parameter"

Reply:
It is appropriate to change to `io/jenkins/plugins/validating-email-parameter`

thanks @slide

Reply:
It is appropriate to change to `io/jenkins/plugins/validating-email-parameter`

thanks @slide

Reply:
You need to make the change, it won't let me make a suggestion or I would have committed it already.

Reply:
I can not commit too. 
Could it be because I reopened the request?

or repository appears as unknown;

<img width="750" alt="Screen Shot 2020-05-15 at 18 28 54" src="https://user-images.githubusercontent.com/16254546/82067866-f32b6b00-96d9-11ea-9ff4-073c4e93e6bd.png">

Reply:
<img width="900" alt="Screen Shot 2020-05-15 at 18 47 17" src="https://user-images.githubusercontent.com/16254546/82069856-98dfd980-96dc-11ea-9c3a-1699fb2ef065.png">